snapshotcreation/snapshotcreation.py

```diff
@@ -24,36 +36,32 @@ def lambda_handler(event, context):
 
     # Iterate through instances identified by the filter.
     for instance in ec2.instances.filter(Filters=event['Filters']):
+        instance_name = instance.instance_id
+        instance_tags = []
+
+        # If a Name tag is available, use it to identify the instance
+        # instead of the instance_id.
+        for tag in instance.tags:
+            if tag['Key'] == 'Name' and tag['Value'] != '':
+                instance_name = tag['Value']
+            else:
+                instance_tags.append(tag)
+
+        try:
+            # Create the AMI
+            image_name = instance_name + '-' + get_unix_timestamp()
```

Comment:
Could we get iso8601 formatted time here instead?  https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat